src/front-end/src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -15,7 +15,6 @@ class ForgotPassword extends Component {
   localResetPassword() {
     // TODO: implement some validation here
     resetPassword(this.state.email);
-    window.alert('We sent you an email with a link to reset your password.');

```

Comment:
Removed this because it would be displayed if an email wasn't entered, but it should only be displayed if a proper email was in the input (see also the updated mySendEmailVerification(user) function in helper-methods)